Test failures with OpenJDK 13

Unit tests are failing when being run under OpenJDK 13 with the following error message:

```
[ERROR] testImputeMissingValuesWithNoMissingValues  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: Internal error: Failed to find the "modifiers" field in method setInternalState.
	at com.amazon.randomcutforest.RandomCutForestTest.setUp(RandomCutForestTest.java:82)
Caused by: java.lang.NoSuchFieldException: modifiers
	at com.amazon.randomcutforest.RandomCutForestTest.setUp(RandomCutForestTest.java:82)
```

```
$ java --version
openjdk 13.0.2 2020-01-14
OpenJDK Runtime Environment (build 13.0.2+8)
OpenJDK 64-Bit Server VM (build 13.0.2+8, mixed mode, sharing)
```

--- a/Java/pom.xml
+++ b/Java/pom.xml
@@ -67,7 +67,7 @@
             <dependency>
                 <groupId>org.powermock</groupId>
                 <artifactId>powermock-api-easymock</artifactId>
-                <version>2.0.2</version>
+                <version>2.0.7</version>
             </dependency>
         </dependencies>
     </dependencyManagement>

